src/admin-views/create-venue-fields.php

```diff
@@ -196,11 +196,11 @@ class="tribe-dropdown"
 	<td>
 		<input
 			tabindex="<?php tribe_events_tab_index(); ?>"
-			type='text'
+			type='url'
 			id='EventWebsite'
 			name='venue[URL][]'
 			size='14'
-			value='<?php echo isset( $_VenueURL ) ? esc_attr( $_VenueURL ) : ''; ?>'
+			value='<?php echo isset( $_VenueURL ) ? esc_url( $_VenueURL ) : ''; ?>'
```

Comment:
@Camwyn this change fixes the issue raised [here](https://theeventscalendar.atlassian.net/browse/TEC-4349?focusedCommentId=105822).

Reply:
Is this safe to do now? I remember [this revert coming](https://github.com/the-events-calendar/the-events-calendar/commit/3b9c62184b89588a9c0948047d4acac81bc64715) through...

Reply:
Sorry, this is referring to this line:
 
```
type='url'
```

Reply:
@stratease Yeah it's working as expected now. The error that made us revert has been fixed by this update. i.e. use 'esc_url' instead of 'esc_attr'

Reply:
Gotcha, perfect thanks!